graphql/rules.lua

```diff
@@ -186,7 +186,13 @@ end
 
 function rules.requiredArgumentsPresent(node, context)
   local arguments = node.arguments or {}
-  local parentField = context.objects[#context.objects - 1].fields[node.name.value]
+  local parentField
+  if context.objects[#context.objects - 1].__type == 'List' then
+    parentField = context.objects[#context.objects - 2].fields[node.name.value]
```

Comment:
So i was kind of "fixing blindly" the errors but i this line above is wrong (i just kind of took a value from somewhere, not the right one)
the line i think should be
```
parentField = context.objects[#context.objects - 1].ofType.fields[node.name.value]
```

i'll add a PR and maybe later some tests